### ISSUE TITLE
Quick Add Keyboard Shortcut not Working

### ISSUE THREAD

**Issue by BWBob** [alainm23/planify#627]
Unable to use the Quick Add key.  Regardless of the shortcut key combo in configuration, no quick add box pops up.

I have viewed at #293 and #263 and was unable to resolve.

**Describe the bug**

1. Load Planner
2. Go to Settings within Planner - Change Quick Add Shortcut key from Super + N (which opens notifications) to Super+T
3. Leave Settings 
4. Move to different program and test new shortcut key.
5. Quick Add box does not appear - nothing happens
6. Have tried different Super and CTRL key combo 

**Expected behavior**
Expect the quick add box will appear when short cut key is used.

**Desktop (please complete the following information):**
Using Linux Mint with Cinnamon Desktop
Planner installed via Software Manager Flatpak
Version 2.5.7

**Comment by alainm23** [comment]
Thank you very much for using Planner, first try to run Quick Add from terminal:
`flatpak run --command=com.github.alainm23.planner.quick-add com.github.alainm23.planner`

- If `Quick Add` works,  in your system go to `Preferences` > `Keyboard` > `Shortcuts` > `Custom Shortcuts`
- Check if the command to run `Quick Add` is available.
- Try removing and configuring it from `Planner`.
- If `Quick Add` cannot be configured from Planner try creating your own custom keyboard shortcut yourself.

**Comment by BeatLink** [comment]
Please let us know if it has been resolved.

**Comment by hojjatabdollahi** [comment]
I tested it. It does not work. 
Here is the custom shortcut that exists in my "custom shortcuts" (It is generated by Planner):
```
Name: flatpak run --command=com.github.alainm23.planner.quick-add com.github.alainm23.planner
Command: flatpak run --command=com.github.alainm23.planner.quick-add com.github.alainm23.planner
Shortcut: Shift+Ctrl+Alt+N
```
If I run the command part in a terminal, the Quick Add window opens.
If I change the command to something else (for example xkill) then pressing `Shift+Ctrl+Alt+N` will run that command. 
But the combination of these two doesn't work. 
I mean having the Quick Add command in the custom shortcuts (no matter what hotkey I use) does not work.

**Comment by nils-asmussen** [comment]
@BWBob uses Cinnamon, which stores the key bindings in a different dconf location than GNOME, e. g.

- GNOME: /org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom0
- Cinnamon: /org/cinnamon/desktop/keybindings/custom-keybindings/custom0

Apart from that, Cinnamon stores the attribute "binding" as **string array,** while GNOME stores a string in "binding"

**Comment by ProfessorCode212** [comment]
> Thank you very much for using Planner, first try to run Quick Add from terminal:
> `flatpak run --command=com.github.alainm23.planner.quick-add com.github.alainm23.planner`

I also run Planner on Linux Mint 19.3 'Cinnamon'. _Quick Add_ didn't work for me either but it worked when I ran that command. 'Removing it and configuring it from planner' **did not work** however so I had to make a custom keyboard shortcut myself. Is there any way to fix this issue permanently?

![image](https://user-images.githubusercontent.com/78950242/108949197-fa7c9d80-7689-11eb-9d3c-550391b59f99.png)

**Comment by alainm23** [comment]
As @nils-asmussen says, `Cinnamon` uses a different dconf location than GNOME, I could do a DE validation and make it work for `Cinnamon` but it will take a little time, I hope I can solve the problem later. Thanks for using Planner.

**Comment by derrickmehaffy** [comment]
FYI I tested the cinnamon custom keybind option and it works great from there: 
![image](https://user-images.githubusercontent.com/8593673/120384218-b88f7300-c2da-11eb-97e9-acb35cc71efb.png)

**Comment by ProfessorCode212** [comment]
@alainm23 _Quick Add_ still doesn't work natively. I still have to use the custom keyboard shortcut I made.

> As @nils-asmussen says, `Cinnamon` uses a different dconf location than GNOME, I could do a DE validation and make it work for `Cinnamon` but it will take a little time, I hope I can solve the problem later. Thanks for using Planner.

Is there an ETA for this? I would very much appreciate it.

**Comment by n3oAlex** [comment]
Just FYI it also doesn't work on XFCE Manjaro, keyboard shortcut is fine for now tho :)

**Comment by hollisticated-horse** [comment]
Same on Gnome Manjaro. Actually none of the shortcuts work...

**Comment by brasilia101** [comment]
@alainm23 This works, thank u

**Comment by harshh100** [comment]
Farsm_se

**Comment by alainm23** [comment]
Hello, Quick Add is available in Planify 4.1, a keyboard shortcut preset is required (Until Global Keyboard portal is available) by the end user. Greetings.

https://flathub.org/es/apps/io.github.alainm23.planify
